packages/sdks/javascript/src/services/UserPasswordService.ts

```diff
@@ -0,0 +1,3 @@
+import { UsersPasswordsApiService } from '../api/generated/apis/users-passwords-api';
+
+export class UserPasswordService extends UsersPasswordsApiService {}
```

Comment:
I believe it would be simpler to just expose a `password` property via the `UserService` - as we do with [users-team-invitations-api](https://github.com/devopness/devopness/blob/main/packages/sdks/javascript/src/services/UserService.ts#L5) - instead of creating a new service.

What do you think?

Reply:
Changed!